deployments/k8s/configmap.yaml

```diff
@@ -54,6 +54,7 @@ data:
     - type: kubelet-metrics
       kubeletAPI:
         authType: serviceAccount
+        url: https://${MY_NODE_NAME}:10250
```

Comment:
Kubernetes has changed this default so many times that we don't really want to specify it here because it is pretty much guaranteed to be wrong for a significant subset of users.